packages/core/src/external/carequality/pd/validating-pd.ts

```diff
@@ -23,7 +23,8 @@ import { STATE_MAPPINGS } from "../shared";
 
 export function validateFHIRAndExtractPatient(payload: InboundPatientDiscoveryReq): PatientData {
   const patient = payload.patientResource;
-  const firstName = patient.name?.[0]?.given?.[0]; // TODO we are taking the first index here but there might be multiple given names
+  //TODO multiple names are not supported yet. We would need to refactor this a bit
+  const firstName = patient.name?.[0]?.given?.join(", ");
```

Comment:
we were just using first given name within the name. Now we make it comma separated and then turn it back into an array in the matching algo